x509-ocsp/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+//! The ocsp module features encoders and decoders for the structures defined in [RFC 6960](https://datatracker.ietf.org/doc/html/rfc6960).
+#![no_std]
+
+pub mod ocsp;
```

Comment:
This module seems somewhat superfluous. I'd suggest putting the contents of `ocsp.rs` in `lib.rs` until such time as there's actually enough code to have more than one module.

Reply:
moved code to lib.rs. I tend to doubt more code lands here, client code ought land elsewhere I think. this is just encoders/decoders.